src/object/cli_obj.c

```diff
@@ -4907,6 +4907,7 @@ obj_dup_sgls_free(struct obj_auxi_args *obj_auxi)
 			d_sg_list_t *sg_orig      = &ctx->sgls_orig[i];
 			uint32_t     dup_sg_idx   = 0;
 			uint32_t     dup_buf_size = 0;
```

Comment:
so the dup_buf_size is only for sanity check after the change, right?

Reply:
Yes.